asset-store/resources/webpage-asset.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: assetstore.kyma-project.io/v1alpha2
+kind: Asset
+metadata:
+  name: webpage
+  namespace: default
+spec:
+  bucketRef:
+    name: pages
+  source:
+    url: https://github.com/flying-seals/page/archive/master.zip
```

Comment:
You are pointing to `flying-seals` repo, but you also attached `index.html` in this repository...